### ISSUE TITLE
Crash when installing any extension

### ISSUE THREAD

**Issue by Camusensei** [tachiyomiorg/tachiyomi#6006]
### Steps to reproduce

Install Tachiyomi
Browse to extensions
Tap "install" on any extension

### Expected behavior

Something asking for installation confirmation should pop up

### Actual behavior

It will hang for 20 seconds, then it will exit the app.
20% of the time, it will un-hang without installing anything instead of existing the app.

### Crash logs

--------- beginning of system
10-02 01:42:27.994 14209 14209 E ActivityThread: Failed to find provider info for downloads
--------- beginning of main
10-02 01:43:11.972 15359 15359 E anade.tachiyom: Not starting debugger since process cannot load the jdwp agent.
10-02 01:45:19.919 15708 15708 E anade.tachiyom: Not starting debugger since process cannot load the jdwp agent.
10-02 01:45:20.257 15708 15753 E SQLiteLog: (284) automatic index on C(manga_id)
10-02 01:45:20.257 15708 15753 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
10-02 01:47:18.205 15938 15938 E anade.tachiyom: Not starting debugger since process cannot load the jdwp agent.
10-02 01:47:22.550 15938 15987 E SQLiteLog: (284) automatic index on C(manga_id)
10-02 01:47:22.550 15938 15987 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
10-02 01:48:34.883 16100 16100 E anade.tachiyom: Not starting debugger since process cannot load the jdwp agent.
10-02 01:48:35.198 16100 16146 E SQLiteLog: (284) automatic index on C(manga_id)
10-02 01:48:35.198 16100 16146 E SQLiteLog: (284) automatic index on mangas_categories(manga_id)
App version: 0.12.2 (standard, 3eec207, 68, 2021-09-13T19:16Z)
Android version: 11 (SDK 30)
Android build ID: RQ3A.210705.001
Device brand: google
Device manufacturer: Google
Device name: sunfish
Device model: Pixel 4a
Device product name: sunfish

### Tachiyomi version

App version: 0.12.2 (standard, 3eec207, 68, 2021-09-13T19:16Z)

### Android version

Android version: 11 (SDK 30) Android build ID: RQ3A.210705.001

### Device

Google Pixel 4a

### Other details

Brand new phone

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.2](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
Does it work if you switch to "Legacy" in Settings > Advanced > Extensions > Installer?

**Comment by Camusensei** [comment]
I don't have "extensions" under settings/advanced (if I had, I'd have tried it already ^^')

**Comment by arkon** [comment]
Whoops, I thought you were using Preview.

**Comment by Camusensei** [comment]
I did as well at some point, let me try again and enable your setting

**Comment by Camusensei** [comment]
Well, it sure is strange but preview works now... I guess I'll use that until the next release ^^

**Comment by Camusensei** [comment]
Even legacy lose works (in preview). Even the stable app works now. Well, I don't know what happened. Maybe a reboot helped, or reinstalling the preview... Anyways thank you!
